package.json

```diff
@@ -19,6 +19,7 @@
   "author": "Santiago Ferreira",
   "license": "MIT",
   "devDependencies": {
-    "qunitjs": "^1.20.0"
+    "qunitjs": "^1.20.0",
+    "testem": "^1.2.0"
```

Comment:
I think `testem` version is not correct. [Looking at `testem` versions](https://github.com/testem/testem/releases) it seems that `1.1.2` is the latest one.

I get this error

```
$ npm install
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/sferreira/.nodenv/versions/4.2.1/bin/node" "/Users/sferreira/.nodenv/versions/4.2.1/bin/npm" "install"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: testem@'>=1.2.0 <2.0.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.0.3","0.0.4","0.0.5","0.0.6","0.0.7","0.0.8","0.0.9","0.0.10","0.0.11","0.0.12","0.0.13","0.0.14","0.0.15","0.0.16","0.0.17","0.0.18","0.0.19","0.0.20","0.0.21","0.0.22","0.0.23","0.0.24","0.0.25","0.0.26","0.0.27","0.0.28","0.0.30","0.0.31","0.0.32","0.0.33","0.0.34","0.0.35","0.0.36","0.0.37","0.0.38","0.0.39","0.0.40","0.0.41","0.0.42","0.0.43","0.0.44","0.0.45","0.0.46","0.0.47","0.0.48","0.0.49","0.0.50","0.0.51","0.0.52","0.0.53","0.0.54","0.0.55","0.0.56","0.0.57","0.1.0","0.1.2","0.1.3","0.1.5","0.1.6","0.1.7","0.1.8","0.2.0","0.2.1","0.2.2","0.2.3","0.2.4","0.2.5","0.2.6","0.2.7","0.2.8","0.2.9","0.2.10","0.2.11","0.2.12","0.2.13","0.2.14","0.2.15","0.2.16","0.2.17","0.2.18","0.2.21","0.2.22","0.2.23","0.2.24","0.2.25","0.2.26","0.2.27","0.2.28","0.2.29","0.2.30","0.2.31","0.2.32","0.2.33","0.2.34","0.2.35","0.2.36","0.2.37","0.2.38","0.2.40","0.2.41","0.2.42","0.2.43","0.2.44","0.2.45","0.2.46","0.2.47","0.2.48","0.2.49","0.2.50","0.2.51","0.2.52","0.2.53","0.2.54","0.2.55","0.2.56","0.2.57","0.2.58","0.2.59","0.2.60","0.2.61","0.2.63","0.2.64","0.2.65","0.2.66","0.2.67","0.2.68","0.2.70","0.2.71","0.2.72","0.2.73","0.2.74","0.2.75","0.2.76","0.2.77","0.2.78","0.2.79","0.2.80","0.2.81","0.2.82","0.2.83","0.2.84","0.2.86","0.2.87","0.2.88","0.2.89","0.2.90","0.2.91","0.2.92","0.2.93","0.2.94","0.2.95","0.2.96","0.2.97","0.2.98","0.3.0","0.3.1","0.3.2","0.3.3","0.3.4","0.3.5","0.3.6","0.3.7","0.3.8","0.3.9","0.3.10","0.3.11","0.3.12","0.3.13","0.3.14","0.3.15","0.3.16","0.3.17","0.3.18","0.3.19","0.3.20","0.3.21","0.3.22","0.3.23","0.3.24","0.3.25","0.3.26","0.3.27","0.3.28","0.3.29","0.3.30","0.4.0","0.4.1","0.4.2","0.4.3","0.4.4","0.4.5","0.4.6","0.4.7","0.4.8","0.5.0","0.5.1","0.5.2","0.5.3","0.5.4","0.5.5","0.5.6","0.5.7","0.5.8","0.5.9","0.5.10","0.5.11","0.5.12","0.5.13","0.5.14","0.5.15","0.6.0","0.6.1","0.6.2","0.6.3","0.6.4","0.6.5","0.6.6","0.6.7","0.6.8","0.6.9","0.6.10","0.6.11","0.6.12","0.6.13","0.6.14","0.6.15","0.6.16","0.6.17","0.6.18","0.6.19","0.6.20","0.6.21","0.6.22","0.6.23","0.6.24","0.6.25","0.6.26","0.6.27","0.6.28","0.6.29","0.6.30","0.6.31","0.6.32","0.6.33","0.6.34","0.6.35","0.6.36","0.6.37","0.6.38","0.6.39","0.7.0-0","0.7.0-1","0.7.0","0.7.1","0.7.2","0.7.3","0.7.4","0.7.5","0.7.6","0.7.7","0.8.0-0","0.8.0","0.8.1","0.8.2","0.8.3","0.8.4","0.8.5","0.9.0-0","0.9.0-1","0.9.0-2","0.9.0","0.9.1","0.9.2","0.9.4","0.9.5","0.9.6","0.9.7","0.9.8","0.9.9","0.9.10","0.9.11","1.0.0-rc.1","1.0.0-rc.2","1.0.0-rc.3","1.0.0-rc.4","1.0.0-rc.5","1.0.0-rc.6","1.0.0-rc.7","1.0.0","1.0.1","1.0.2","1.1.0","1.1.1","1.1.2"]
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'ceibo'
npm ERR! notarget

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/sferreira/Work/Personal/ceibo/npm-debug.log
```